src/components/templates/Jazzbar/JazzTab/JazzTab.tsx

```diff
@@ -58,43 +66,39 @@ const Jazz: React.FC<JazzProps> = ({ setUserList, venue }) => {
     },
   ]);
 
-  const [nowMs, setNowMs] = useState(Date.now());
-
-  useInterval(() => {
-    setNowMs(Date.now());
-  }, LOC_UPDATE_FREQ_MS);
-
   const { user } = useUser();
 
   const firestoreVenue = useSelector(currentVenueSelectorData);
-  const users = useSelector(partygoersSelector);
 
   const venueToUse = venue ? venue : firestoreVenue;
 
   const jazzbarTables = venueToUse?.config?.tables ?? JAZZBAR_TABLES;
 
-  const venueUsers = users
-    ? users.filter(
-        (user) =>
-          !!user.lastSeenIn &&
-          user.lastSeenIn[venueToUse?.name ?? ""] >
-            (nowMs - LOC_UPDATE_FREQ_MS * 2) / 1000
-      )
-    : [];
+  // TODO: this will break memo on venueUsers (below) every 5min, does that matter?
+  const [nowMs, setNowMs] = useState(Date.now());
+  useInterval(() => {
+    setNowMs(Date.now());
+  }, LOC_UPDATE_FREQ_MS);
```

Comment:
Consider wrapping this in `useEffect` so we can undo it on unmount. Although, it is likely unimportant as this component has the same lifecycle as the page view.

Reply:
`useInterval` is a wrapping hook i wrote that handles all of that for us in a single reusable way. It clears the interval on unmount:

- https://github.com/sparkletown/sparkle/blob/staging/src/hooks/useInterval.ts

Reply:
No problem, I see that now.